src/pages/content/punctuation.mdx

```diff
@@ -1,12 +1,97 @@
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Breadcrumbs              | No. Bullets should be sentence fragments. If a bullet has more than one sentence, use closing punctuation on each. If a bullet has more than two sentences, it should be a description instead. |
+| Bullets (in a list)      | No                                                                                                                                                                                              |
+| Button                   | No                                                                                                                                                                                              |
+| Description              | Yes, at the end of every full sentence.                                                                                                                                                         |
+| Headings and subheadings | No                                                                                                                                                                                              |
+| Labels                   | No                                                                                                                                                                                              |
+| System messages          | No, unless it's more than one sentence.                                                                                                                                                         |
+| Title                    | No                                                                                                                                                                                              |
+| Tooltip                  | No, unless it's more than one sentence.                                                                                                                                                         |
+
+## Quotation mark
+
+Avoid using quote marks to indicate places and features in the UI. Use bold instead.
+
+<BestPractice>
+  <Do>
+    <p>Use the End chat button to end a conversation.</p>
+  </Do>
+  <Dont>
+    <p>Use the “End chat” button to end a conversation.</p>
+  </Dont>
+</BestPractice>
+
+## Semi-colon
+
+No semi-colons in product; that's the way it has to be.
+
+## Slash
+
+Used when indicating URLs and as a date separator. You can also use slashes to mean “per,” but limit
+use to one slash in a phrase, for example: “\$9/agent per month.”
```

Comment:
Is the backslash a copy bug?

Reply:
Yeah, I'm a little unsure how that ended up there.

Reply:
Prettier is adding it

Reply:
Appears to be some errant regex treatment 🙁 You might need to resort to using `prettier-ignore` 🤦